src/views/home/HomeView.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { Topbar, BasicInformation, TilesSection, RecommendedTemplates, HomeDescription, Footer } from '.';
+
+const HomeView = () => {
+  return (
+    <>
```

Comment:
You should add html tag wrapper. <div className={csx.homeView}>{CURRENT CONTENT}</div. Using react fragment to page layout is risky